### PR TITLE
More twisted tox versions + codecov

### PR DESCRIPTION
**Pull request by hawkowl**

### CODE REVIEW

**Comment by hawkowl** [comment]
cc @oberstet, this is ready for review

File: tox.ini [hunk 1]
```diff
@@ -1,180 +1,47 @@
 [tox]
-envlist = flake8,
-          py26twisted,
-          py26asyncio,
```
Comment (oberstet) [inline tox.ini]: isn't this missing after the change?

Comment (hawkowl) [inline tox.ini]: gah yes
I wanted to remove py26-twtrunk, as Twisted trunk no longer supports 2.6. Will fix.

File: tox.ini [hunk 2]
```diff
@@ -1,180 +1,47 @@
...
-   twisted
-   unittest2
-commands =
-   sh -c "which python"
-   sh -c "which trial"
-   python -V
-   trial --version
-   trial autobahn
-basepython = python2.6
-install_command = pip install {packages} autobahn[twisted,serialization]
```
Comment (oberstet) [inline tox.ini]: applies to all envs: `serialization` is an extra that eg installs msgpack. without that, msgpack WAMP serialization won't be tested (only JSON). is that extra install dep lost?

Comment (hawkowl) [inline tox.ini]: Yep, it installs it explicitly at line 42.

File: tox.ini (hunk 1 above)
Comment (hawkowl) [inline tox.ini]: OK, `py26-trollius` is restored.

File: tox.ini (hunk 2 above)
Comment (hawkowl) [inline tox.ini]: Or rather, no, it's not lost, it's installed explicitly at line 42.

Comment (oberstet) [inline tox.ini]: I'm too dumb .. where is it? https://github.com/hawkowl/AutobahnPython/blob/more-twisted-tox-versions/tox.ini#L42 doesn't have it ..

Comment (hawkowl) [inline tox.ini]: Erm, seems like I'm the dumb one here actually -- this should be the URL: https://github.com/hawkowl/AutobahnPython/blob/more-twisted-tox-versions/tox.ini#L14

**Comment by hawkowl** [comment]
Okay, travis (with `py26-trollius`) seems to have passed fine @oberstet

File: tox.ini (hunk 2 above)
Comment (oberstet) [inline tox.ini]: Ahh, right. That's fine As long as we keep that in sync with https://github.com/tavendo/AutobahnPython/blob/master/setup.py#L123

**Pull request merged**